spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/services/SCBackendService.java

```diff
@@ -54,8 +54,18 @@
     private Context context;
     private MqttHandler mqttHandler;
     private Observable<SCNotification> notifications;
+    /**
+     * Behavior Observable emitting YES when the SpatialConnect SCConfig has been received
```

Comment:
Can you change `YES` to `true` and `NO` to `false` to be more explicit about what is being emitted?